### PR TITLE
Example/A person jumping

### PR DESCRIPTION
**Pull request by Kartik-Sama**
Your branch is up to date with 'origin/master'.
This example uses canvas to show a jumping person.
Changes to be committed:
	modified:   index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,26 +7,67 @@
 
 </html>
 <script>
-
     // Declare all global variables here
```
Comment (satu0king) [inline index.html:11]: Add your name as author here.

**Review by satu0king — CHANGES_REQUESTED** [review]
Add your name and we can merge.

File: index.html [hunk 2]
```diff
@@ -7,26 +7,67 @@
 
 </html>
 <script>
-
```
Comment (Kartik-Sama) [inline index.html]: //Work by Karthik Sama

File: index.html (hunk 1 above)
Comment (Kartik-Sama) [reply index.html]: I have added a comment. Hope this is what you meant.

Comment (Kartik-Sama) [reply index.html:11]: In my recent commit I have added my name. Hope this is what you asked for.

Comment (satu0king) [reply index.html:11]: Yes but more like 
`Author - Kathik Sama`
I will merge for now

**Pull request merged**